css/style.css

```diff
@@ -50,3 +58,56 @@ font-family: 'Raleway', sans-serif;
   font-size: 60px;
   border-bottom: #487e95 2px solid;
 }
+
+
+/* Start Gallery Section */
+
+.gallery {
+    margin: 20px;
+    text-align: center;
+}
+
+.gallery h2 {
+    color: #487e95;
+    font-size: 3.5vw;
+    font-weight: bold;
+}
+
+.row {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 0 4px;
+}
+  
+.column {
+    flex: 32%;
+    max-width: 33%;
+    padding: 0 6px;
+}
+  
+.column img {
+    margin-top: 10px;
+    vertical-align: middle;
+    width: 100%;
+    border-radius: 5%;
+}
+
+.my-button {
+	background: none;
+	border: 2px solid #487e95;
+    border-radius: 30px;
+	padding: 10px 30px;
+	color: #487e95;
+	font-weight: bold;
+    margin-top: 20px;
+	font-size: 1.5vw;
+    outline: none;
+}
```

Comment:
pay attention to your code formatting, use prettier correctly

![Peek 2021-03-02 19-15](https://user-images.githubusercontent.com/47992412/109871103-7f138100-7c73-11eb-92d7-e9bd568d551a.gif)